components/raftstore-v2/src/fsm/peer.rs

```diff
@@ -196,6 +196,7 @@ impl<'a, EK: KvEngine, ER: RaftEngine, T: Transport> PeerFsmDelegate<'a, EK, ER,
         self.schedule_tick(PeerTick::SplitRegionCheck);
         self.schedule_tick(PeerTick::PdHeartbeat);
         self.schedule_tick(PeerTick::CompactLog);
+        self.fsm.peer.on_check_merge(self.store_ctx);
```

Comment:
How about adding `CheckMerge` tick? (to be consistent with others)
```suggestion
        self.schedule_tick(PeerTick::CheckMerge);
```

Reply:
Using schedule_tick delays the check for some time. I prefer to check immediately after restart.